fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -2365,7 +2365,6 @@ baseType
     | DOUBLE
     | DATE
     | DATETIME
-    | TIME
     | CHAR typeParameter?
```

Comment:
Now, cast(value as time) is supported. Maybe add check in DDL is better?
mysql> select cast(100 as time);
+-------------------+
| CAST(100 AS TIME) |
+-------------------+
| 00:01:00          |
+-------------------+
1 row in set (0.01 sec)

Reply:
ok